src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { useContext, useMemo } from 'react';
+import classNames from 'classnames';
+import { TypeOfFiltering } from '../../types/TypeOfFiltering';
+import { TodoContext } from '../../contexts/TodoContext';
+
+export const Footer:React.FC = () => {
+  const { todos, deleteData } = useContext(TodoContext);
+  const { filterType, setFilterType } = useContext(TodoContext);
+
+  const notCompletedTodosCount = useMemo(() => {
+    return todos.filter(todo => !todo.completed).length;
+  }, [todos]);
+
+  const addS = () => {
+    if (notCompletedTodosCount !== 1) {
+      return 's';
+    }
+
+    return 's';
+  };
```

Comment:
looks like it will return `s` in both cases

Reply:
it shouldn't be like that, need to investigate this question

Reply:
https://github.com/mate-academy/react_todo-app-with-api/pull/1115#issuecomment-2018410499